lib/manager/travis/readme.md

```diff
@@ -1,7 +1,7 @@
 This manager is intended to keep Travis config files (`.travis.yml`) up-to-date, this file controls the CI build environment.
 Currently Renovate can only update the `node_js` section of this file.
 
-An important limitation is that Renovate does not currently "understand" [Travis's Build Matrix concept](https://docs.travis-ci.com/user/build-matrix/#matrix-expansion), so it will try to update all found Node.js versions to the latest LTS, e.g.
+Renovate "understands" [Travis's Build Matrix concept](https://docs.travis-ci.com/user/build-matrix/#matrix-expansion) as well, so it will try to update all found Node.js versions to the latest LTS, e.g.
```

Comment:
Can you please edit the the "diff" example below this suggestion (lines 8 through 11), Node `16.13.0` is the latest LTS now. 😉 

Is the "major updates are disabled by default" section still accurate after this PR gets merged?

```markdown
Due to this, major updates for Travis are disabled by default.
If you enable major updates and use a version matrix, then you will likely need to manually fix any major update PRs raised by Renovate.
Here's how to enable major updates in your Renovate config:

{
  "travis": {
    "major": {
      "enabled": true
    }
  }
}
```

Reply:
Hi @HonkingGoose, regarding "major updates are disabled by default" I think that section should be removed, but I'm not sure. I'd ask @viceice for advice.

Reply:
This still applies:

https://github.com/renovatebot/renovate/blob/a2ceff099f4e63807ac99084e64c9a0203f961c7/lib/manager/travis/index.ts#L10-L12